stable/heapster/Chart.yaml

```diff
@@ -1,7 +1,9 @@
 apiVersion: v1
 description: Heapster enables Container Cluster Monitoring and Performance Analysis.
 name: heapster
-version: 0.2.8
+version: 0.2.9
+appVersion: 1.3.0
+home: https://www.heapster.net/
```

Comment:
I don't think this is the heapster you're looking for. The github link used in sources is the definitive home, I think.

Reply:
Thanks @timstoop ,Do you mean we should use the following URL? I've changed to it 
home: https://github.com/kubernetes/heapster